designer/processReports/src/main/scala/pl/touk/nussknacker/processCounts/influxdb/InfluxGenerator.scala

```diff
@@ -62,8 +62,6 @@ private[influxdb] class InfluxGenerator[F[_]](config: InfluxConfig, env: String)
   private def parseInfluxDate(date:String) : Instant =
     ZonedDateTime.parse(date, DateTimeFormatter.ISO_ZONED_DATE_TIME).toInstant
 
-  def close(): F[Unit] = influxClient.close()
```

Comment:
`SimpleInfluxClient` has a close method so it looks like we need to remove it too. Especially, the method implementation is wrong - it closes the implicitly passed `backend` (it's not an owner of the resource, so it should not bother stopping it).

Reply:
ok. I didn't notice changes in SimpleInfluxClient